includes/payment-methods/class-afterpay-payment-method.php

```diff
@@ -72,8 +72,10 @@ public function __construct( $token_service ) {
 	 * Returns payment method title.
 	 *
 	 * @param string|null $account_country Country of merchants account.
-	 * @param array|false $payment_details Optional payment details from charge object.
+	 * @param array|false $payment_details Payment details from charge object. Not used by this class.
 	 * @return string|null
+	 *
+	 * @phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
 	 */
 	public function get_title( string $account_country = null, $payment_details = false ) {
```

Comment:
Is this class inherited somewhere? If not, the recommended fix here would be to prepend `$_unused_` to the variable name instead of using a phpcs ignore:

```suggestion
	 * @param array|false $payment_details Optional payment details from charge object.
	 * @param array|false $_unused_payment_details Payment details from charge object. Not used by this class.
	 * @return string|null
	 */
	public function get_title( string $account_country = null, $_unused_payment_details = false ) {
```

This also applies to the other 2 ignores added in this PR.

Reply:
@reykjalin As per https://github.com/Automattic/woocommerce-payments/pull/8955/files#r1638396764, prepending the variable name with `$_unused` leads to [the `ParamNameMismatch`](https://github.com/Automattic/woocommerce-payments/actions/runs/9501032020/job/26185593016?pr=8955) which is turn can be fixed by another suppression but doesn't seem to be worth it, so I applied the ignore. Let me know your thoughts, thanks!

Reply:
Hmmm, I don't think it matters if the doc comment is changed to be `$_unused_<var_name>` tbh, and that would fix that lint failure. I'll modify my suggestion to demonstrate 👍 

> If you're referring to `get_title` and `get_icon`, both methods are located in `includes/payment-methods/class-upe-payment-method.php` which is inherited by payment method classes, so the ignores were applied here as per instructions.

Awesome, just wanted to make sure, thank you!

Reply:
oh, wait no that's a Psalm failure! Huh, didn't think about that. Let's keep this as is for now, I'll open an issue to track how we address this.

Reply:
I've opened https://github.com/Automattic/woocommerce-payments/issues/8956 to track this and added instructions to use phpcs ignores for child classes to all the remaining issue descriptions 👍